frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx

```diff
@@ -25,9 +25,16 @@ type Props = NodeProps<TableNodeType>
 
 export const TableNode: FC<Props> = ({ data: { table } }) => {
   const { relationships } = useDBStructureStore()
+  const [isOpen, setIsOpen] = useState(false)
+
   return (
     <>
-      <DrawerRoot direction="right">
+      <DrawerRoot
+        direction="right"
+        dismissible={false}
```

Comment:
see: https://vaul.emilkowal.ski/other#non-dismissible

Reply:
The close button is far away, so it looked good on dismissible, should we set it to false?

Reply:
nice comments 👍🏻